markdown/3.x/en/faq.md

```diff
@@ -3,7 +3,21 @@
 #### Question:
 I want to use FACT-Finder Web Components with Angular, but data binding used in the components conflicts with Angular's `{{ }}` binding. Can you fix it?
 #### Answer:
-Default binding delimiters can be changed via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine).
+You can add Angular's `ngNonBindable` attribute to the parent element to prevent Angular from parsing double curly braces. If you have to keep both Angular and FACT-Finder WebComponents bindings within the same element, you can use: 
+```html
+<ng-container ngNonBindable>
+    {{FF Binding}}
+</ng-container>
```

Comment:
This example is fine, but taken from the context it is not clear. I guess `<ng-container ngNonBindable>` should wrap some FACT-Finder WebComponent with template, like:
```
<div>
    <!-- Any bindings here will be handled by Angular -->
    <ng-container ngNonBindable>
        <ff-template scope="result">
            <!-- This binding will be ignored by Angular and handled by ff-template  -->
            <span>Product found: {{resultCount}}</span>
        </ff-template>
    </ng-container>
</div>```

Reply:
Actually, it shouldn't :D I'll extend the examples then.